### DISCUSSION TITLE
Is it possible to remove/delete a package from universe ?

### DISCUSSION THREAD

**Discussion by ghdc-ispm** [r-universe-org/help#471] [Q&A] [answered]
And the universe itself ? And how ?

Thank you

**Comment by jeroen** [comment]
If you set up your own universe you can control which packages are included via your `packages.json` file.

If you don't have any custom universe but only some CRAN packages, you can set `Config/runiverse/noindex: true` in your package `DESCRIPTION` to prevent it from being scraped. See https://docs.r-universe.dev/publish/set-up.html#how-to-prevent-a-cranbioc-package-from-automatically-being-indexed-by-r-universe

**Reply by ghdc-ispm** [reply]
Indeed, I got suggestion to use r-universe to facilitate package building and installation (the package is on GitHub repo) in order to share with non-technical team members, without the need to install the dev tool. But I do not want to "publish" it as a public package for everyone yet. So is it the way to do: following your instruction to setup the universe and include the package, but prevent it to be indexed ?

**Reply by jeroen** [reply]
r-universe is a public platform, so if you publish your packages on there, they will be visible for other people as well. There is no way to make it private to your team only.

**Reply by ghdc-ispm** [reply]
OK, in case we want to remove published package and add new ones, it will also take like 1h for the universe to adopt the changes ? And to my original question, is it possible to delete a universe once we created it ? Thank you

**Reply by jeroen** [reply, marked as answer]
Yes it takes one hour. If you delete the packages.json then the universe gets deleted.
